compact/materials.xml

```diff
@@ -334,6 +334,25 @@
     <fraction n="0.528" ref="Quartz"/>
     <fraction n="0.472" ref="Fr4_Epoxy"/>
   </material>
+  <material name="Tyvek">
+    <D type="density" value="0.3529" unit="g/cm3"/>
+    <fraction n="0.528" ref="Quartz"/>
+    <fraction n="0.472" ref="Fr4_Epoxy"/>
+  </material>
```

Comment:
Is tyvek really a mixture of quartz and epoxy???

Reply:
Tyvek is very similar in its composition to a PCB just lower density (that's why I went with that.)

Reply:
Tyvek is not similar in composition to PCBs and it for sure is not > 50% quartz (or glass); there is no Si at all in tyvek. Please use the correct composition. When you make Tyvek available as a material, others may use it assuming it is at least somewhat correct.